static/app/views/performance/landing/widgets/widgets/slowScreens.tsx

```diff
@@ -0,0 +1,396 @@
+  const [selectedListIndex, setSelectListIndex] = useState<number>(0);
+  const {organization, InteractiveTitle} = props;
+  const pageError = usePageError();
+
+  const field = props.fields[0];
+
+  const listQuery = useMemo<QueryDefinition<DataType, WidgetDataResult>>(
+    () => ({
+      fields: field,
+      component: provided => {
+        const eventView = provided.eventView.clone();
+        let extraQueryParams = getMEPParamsIfApplicable(mepSetting, props.chartSetting);
+
+        // Set fields
+        eventView.fields = [
+          {field: 'transaction'},
+          {field: YAXIS_COLUMNS[YAxis.TTID]},
+          {field: 'count()'},
+        ];
+        eventView.sorts = [
+          {
+            field: 'count()',
+            kind: 'desc',
+          },
+        ];
+
+        // Change data set to metrics
+        eventView.dataset = DiscoverDatasets.METRICS;
+        extraQueryParams = {
+          ...extraQueryParams,
+          dataset: DiscoverDatasets.METRICS,
+        };
+
+        // Update query
+        const mutableSearch = new MutableSearch(eventView.query);
+        mutableSearch.addFilterValue('event.type', 'transaction');
+        mutableSearch.addFilterValue('transaction.op', 'ui.load');
+        eventView.query = `${mutableSearch.formatString()} release:[${primaryRelease},${secondaryRelease}]`;
+
+        return (
+          <DiscoverQuery
+            {...provided}
+            eventView={eventView}
+            location={location}
+            limit={QUERY_LIMIT_PARAM}
+            cursor="0:0:1"
+            noPagination
+            queryExtras={extraQueryParams}
+          />
+        );
+      },
+      transform: transformDiscoverToList,
+    }),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [props.chartSetting, mepSetting.memoizationKey, primaryRelease, secondaryRelease]
+  );
+
+  const chartQuery = useMemo<QueryDefinition<DataType, WidgetDataResult>>(
+    () => {
+      return {
+        enabled: widgetData => {
+          return !!widgetData?.list?.data?.length;
+        },
+        fields: field,
+        component: provided => {
+          const eventView = props.eventView.clone();
+          let extraQueryParams = getMEPParamsIfApplicable(mepSetting, props.chartSetting);
+          const pageFilterDatetime = {
+            start: provided.start,
+            end: provided.end,
+            period: provided.period,
+          };
+
+          // Chart options
+          const currentSeriesNames = [field];
+          const includePreviousParam = true;
+          const yAxis = provided.yAxis;
+          const interval = getInterval(pageFilterDatetime, 'medium');
+          const partialDataParam = true;
+
+          if (
+            !provided.widgetData.list.data[selectedListIndex] ||
+            (!provided.widgetData.list.data[selectedListIndex]?.transaction &&
+              !provided.widgetData.list.data[selectedListIndex][
+                SpanMetricsField.SPAN_DESCRIPTION
```

Comment:
Not quiteeee sure what this is checking, we're not hitting span metrics, so we shouldn't be checking any span description things

Reply:
Copy pasta mistake 🙈 Removed.